tap/tracker.py

```diff
@@ -139,6 +148,14 @@ def generate_tap_report(self, test_case, tap_lines, out_file):
         if not self.combined:
             print('1..{0}'.format(len(tap_lines)), file=out_file)
 
+    def _write_tappy_version(self, filename):
+        """
+        Writes a version header row if Version 13 of Tappy is enabled via
```

Comment:
I usually try to follow PEP 257 conventions for docstrings. Roughly the format is like:
```
"""Do something (an imperative sentence describing what the function does).

Here are the additional details that might be relevant to know about.
"""
```
https://www.python.org/dev/peps/pep-0257/